hugo/content/documentation/administration/acl/index.md

```diff
@@ -0,0 +1,250 @@
+
+| Prefix | Prefix-Type   | Description                                      | Example                        |
+| ------ | ------------- | ------------------------------------------------ | ------------------------------ |
+| *none* | Id-Type       | User                                             | `User1`, `John Doe`            |
+| `@`    | Id-Type       | Group                                            | `@admin`, `@all`, `@moderator` |
+| `@#`   | Id-Type       | Channel password / Access Token                  | `@#secret`, `@#u8DhwG2`        |
+| `~`    | Eval-Locality | Evaluate in context of definition, not inherited | `@~sub 1,1`, `@~moderator`     |
+| `!`    | Inversion     | Invert membership (not in group)                 | `!TrustedUser`, `!@moderator`  |
+
+The tilde `~` eliminates any effect of inheritance or links.
+
+### Permission Allow/Deny
+
+The permission actions can allow or deny a permission. If the rule neither allows nor denies a permission it does not influence the top-to-bottom permission evaluation.
+
+### Rule Examples
+
+For example, given the following channel setup
+
+```text
+Root
+  ChanA1
+    ChanA11
+    ChanA12
+    ChanA13
+  ChanB
+```
+
+`@admin` defined on channel `Root` with *Allow* `Write ACL` will give members of the `admin` group full permissions.
+
+![Screenshot of giving admin group Write ACL permission](acl-admin-allow-writeacl.jpg)
+
+### Special Groups
+
+Some special groups can be used in ACL rules. The members of these groups are not defined explicitly. They are implicit - a consequence of the user state.
+
+| Group    | Description         | Notes          |
+| -------- | ------------------- | -------------- |
+| `@all`   | Applies to everyone                  | Useful for default permissions or for clearing complex rules to a well defined state. |
+| `@admin` | People with administrative authority |  |
+| `@in`    | In this channel                      |  |
+| `@out`   | Outside of this channel              |  |
+| `@sub`   | Sub-channel - `@sub,a,b,c`           |  |
+
+#### `in` Group
+
+Given a channel tree
+
+```text
+Root
+  ChanA1
+    ChanA11
+    ChanA12
+  ChanB
+```
+
+To deny users outside of ChanA1 to send text messages to it, but allow users inside of it to send text messages, we add two rules.
+
+* *Deny* *Text message* to `@all`
+* *Allow* *Text message* to `@in`
+
+Now users in `Root`, `ChanA11`, `ChanA12`, ChanB can not send text messages to ChanA1. But users inside of ChanA1 can.
+
+![Screenshot of in rule example all rule](acl-in-example1-1all.jpg)
+
+![Screenshot of in rule example in rule](acl-in-example1-2in.jpg)
+
+The rule is inherited into `ChanA11` and `ChanA12`. But it applies as a rule within that channel. So users outside of `ChanA11` can not send text messages to `ChanA11`, but users inside of it can. Users outside of `ChanA12` can not send text messages to `ChanA12`, but users inside of it can.

```

Comment:
```suggestion
The rule is inherited into `ChanA11` and `ChanA12`. But it applies as a rule within each of those  channels individually. So users outside of `ChanA11` can not send text messages to `ChanA11`, but users inside of it can. Users outside of `ChanA12` can not send text messages to `ChanA12`, but users inside of it can.
```